active/fundraising.md

```diff
@@ -2,31 +2,60 @@
 
 ## Scope of responsibilities
 
-Coordinates fundraising efforts, particularly around corporate and major donations.
+This working group coordinates fundraising efforts, particularly around corporate and major donations.
+
+The goals of the group are:
+
+- Acquire funding to secure the financial future of Django and support future growth of Django
+- Build and strengthen relationships between Django and donor sources
+- Raise awareness of the Django Web Framework and its community with the broader world
+
+Delegated responsibilities:
+
+- Contacting, developing and implementing financial sponsorships with donor sources
+- Clarifying and elaborating on the various creative strategies that donor sources of different sizes and types can use to support Django
 
-Delegated responsibilities TBD.
 
 ## Initial membership
 
 - Chair: TBD
 - Co-Chair: TBD
 - Board Liaison (must be an active Board member; may be the same as Chair/Co-Chair): TBD
-- Other members: TBD
+- Other members:
+  - Jacob Kaplan-Moss
+  - Priya Pahwa
+  - Thibaud Colas
+  - Tom Carrick
+  - Sarah Abderemane
+  - Stephanie Goulet
+
 
 ## Future membership
 
 Direct membership: new members may self-nominate; the WG will vote (50%+1) to approve/deny new members; the WG will directly vote on new Chair/Co-Chairs.
 
-This committee is currently forming and seeking volinteers, including chair and co-chair. 
+This committee is currently forming and seeking volunteers, including chair and co-chair. 
+
+### Navigating Conflicts of Interest
+Each new member must disclose any potential conflicts of interest. These disclosures will be made transparent by means of ____. 
```

Comment:
I'm unsure what is the best way, initially I thought about an email google group but could be something else

Reply:
I’d expect it’d be more useful to disclose conflicts of interest as part of decision-making on a case by case basis, than upfront?

I’m not sure I see why this needs to be part of the WG charter to start with, it’s important but even for the board I don’t believe we have this so formally defined.

Reply:
Yeah, I do think it's fine to leave the specific mechanism of COI disclosure out of the charter -- it can be an operating procedure, not something in the charter. 

That said, I think it might be sufficient to just ask for COI disclosures as part of the application process, in which case you can easily just add that there. Something like this could work:

> When new members self-nominate, they should disclose any potential COIs to the WG. They should also disclose any COIs that might affect a vote before that vote